components/config-util/pom.xml

```diff
@@ -2,7 +2,7 @@
     <parent>
         <groupId>com.wso2telco.core</groupId>
         <artifactId>core-parent</artifactId>
-        <version>2.4.5-SNAPSHOT</version>
+        <version>2.4.6-SNAPSHOT</version>
```

Comment:
Can you remove the dependency version. Since we are referring to the parent pom version will be picked from there. No need to specify version tag

Reply:
My bad, this is not a dependency right. Will resolve this. Nothing to fix on this.